app/utils/pagerDutyLogging/PagerDutyLogging.scala

```diff
@@ -19,10 +19,19 @@ package utils.pagerDutyLogging
 object PagerDutyLogging {
   def logError(loggerMessage: LoggerMessages.Value, status: Int, body: String,
```

Comment:
Maybe rename method to just `log` (rather than `logError`) since it is not necessarily logging as error level.